.github/workflows/merge_to_master.yml

```diff
@@ -0,0 +1,70 @@
+# CI stages to execute against master branch on PR merge
```

Comment:
This workflow will run against all branches not just master. We should update this comment and name of this yml file.

Reply:
@jameerpathan111 Updated.